sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -629,6 +629,64 @@
     "hd_job"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3. Automatic hyperparameter sweeping for your AutoML models for computer vision tasks (Automode)\n",
+    "\n",
+    "When using AutoML for Images, we can perform an automatic hyperparameter sweep to find the optimal model. This automation functionality, which we call Automode, will choose a model architecture and values for the learning_rate, number_of_epochs, training_batch_size, etc. based on the number of runs. There is no need to specify the hyperparameter search space, sampling method or early termination policy. A number of runs between 10 and 20 will likely work well on many datasets.\n",
+    "\n",
+    "Automode is triggered by setting `max_trials` to a value greater than 1 in limits and by omitting the hyperparameter space, sampling method and termination policy."
```

Comment:
Assuming that we have some explanation/guidance on appropriate values to play around for max_trails in our docs.

Reply:
Yes.